Program.cs

```diff
@@ -2,6 +2,7 @@
 
 // Add services to the container.
 builder.Services.AddRazorPages();
+builder.Services.AddHttpClient();

```

Comment:
This will manage the lifetime of http clients and inject them into your controllers.